src/fides/api/ops/schemas/connection_configuration/__init__.py

```diff
@@ -80,6 +83,15 @@
 from fides.api.ops.schemas.connection_configuration.connection_secrets_snowflake import (
     SnowflakeSchema as SnowflakeSchema,
 )
+from fides.api.ops.schemas.connection_configuration.connection_secrets_sovrn import (
+    SOVRN_REQUIRED_IDENTITY as SVORN_REQUIRED_IDENTITY,
```

Comment:
whoops looks like one `SVORN` stuck around :) 

```suggestion
    SOVRN_REQUIRED_IDENTITY as SOVRN_REQUIRED_IDENTITY,
```